src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
This variable should not be static, each instance of ColorSupplier should have its own instance of Random class